From 667e477cfaa54b3e0609a8b665da63144393346d Mon Sep 17 00:00:00 2001
From: Paul Sauve <paul@technove.co>
Date: Mon, 29 Mar 2021 14:35:37 -0500
Subject: [PATCH] Disable Downloading Skins for Skulls


diff --git a/src/main/java/net/minecraft/server/TileEntitySkull.java b/src/main/java/net/minecraft/server/TileEntitySkull.java
index dd2665881..4fe71cf34 100644
--- a/src/main/java/net/minecraft/server/TileEntitySkull.java
+++ b/src/main/java/net/minecraft/server/TileEntitySkull.java
@@ -182,7 +182,7 @@ public class TileEntitySkull extends TileEntity /*implements ITickable*/ { // Pa
                 }
                 return false;
             }
-        }, false); 
+        }, false);
         // Spigot end
     }
 
diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaSkull.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaSkull.java
index 1ac549e5e..d0ab100ec 100644
--- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaSkull.java
+++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaSkull.java
@@ -83,7 +83,11 @@ class CraftMetaSkull extends CraftMetaItem implements SkullMeta {
         if (profile != null) {
             // Fill in textures
             // Must be done sync due to way client handles textures
+            // Paul start - disable lookups
+            /*
             profile = com.google.common.util.concurrent.Futures.getUnchecked(TileEntitySkull.b(profile, com.google.common.base.Predicates.alwaysTrue(), true)); // Spigot
+             */
+            // Paul end
 
             NBTTagCompound owner = new NBTTagCompound();
             GameProfileSerializer.serialize(owner, profile);
-- 
2.31.0

